### PR TITLE
AI doc update - Checkbox & Radio button

### LINKED ISSUE #4009: AI documentation: Add AI docs for Checkbox
Update the checkbox component page using the [AI component guidelines template](https://github.com/carbon-design-system/carbon-website/issues/4007).

## Usage tab
[See example in Figma](https://www.figma.com/file/OJwBWBlctEgm9zYUCYbhu3/Carbon-for-AI-%E2%80%93-website-integration?type=design&node-id=80%3A7220&mode=design&t=XPoeQ0K8Ea0VbhbW-1)

- [x] Add inline notification to the top of the page under intro paragraph. The link should go to the AI component’s storybook example.
- [x] Add “AI presence” to the anchor links.
- [x] Add an H2 section for “AI presence” between “Modifiers” (or “Behaviors”) and “Related” content sections.
- [x] Copy the content from the template example and edit the template content to be specific to each component. Only the input components will get the extra revert h3 sections.
- [x] Create images for the components similar to the template examples (add images to the [Usage and Style images](https://www.figma.com/file/9J3QEJ717fLUa7tzlWVMMu/Usage-and-Style-images?type=design&t=VpbAnlbxwfthmkMr-6) file)

_If links to the general AI guidelines sections aren’t ready yet then don’t include links or hyperlinks yet. We will add these in later when ready._

## Style
[See example in Figma](https://www.figma.com/file/OJwBWBlctEgm9zYUCYbhu3/Carbon-for-AI-%E2%80%93-website-integration?type=design&node-id=80%3A7192&mode=design&t=XPoeQ0K8Ea0VbhbW-1)

- [x] Add “AI presence” to the anchor links.
- [x] Add an H2 section for “AI presence” at the end of the page
- [x] Copy the content from the template example and edit the template content where needed to be specific to each component.
- [x] Create an image for the component similar to the template examples. (add images to the [Usage and Style images](https://www.figma.com/file/9J3QEJ717fLUa7tzlWVMMu/Usage-and-Style-images?type=design&t=VpbAnlbxwfthmkMr-6) file)

_If links to the general AI guidelines sections aren’t ready yet then don’t include links or hyperlinks yet. We will add these in later when ready._

## Accessibility 
No changes needed at the moment.

---

# Deliverables
- [x] Pull request opened in the website repo with content updates

### PR DESCRIPTION
**Pull request by Kritvi-bhatia17**
Closes #4009 & #4037 

### Added in the Usage tab
- Inline notification
- AI Presence Section

### Added in the Style tab
- AI Styling table

(Bug fixes:)
- Page description + Anchor links
- Feedback section

### CODE REVIEW

File: src/pages/components/checkbox/style.mdx [hunk 1]
```diff
@@ -101,3 +118,38 @@ labels should not exceed three words.
...
+
+More information about designing for AI guidelines is coming soon.
+
+| Element         | Property         | Token / Size        |
+| --------------- | ---------------- | ------------------- |
+| Linear-gradient | start            | `$ai-aura-start-sm` |
+|                 | stop             | `$ai-aura-stop`     |
+| Field           | border-bottom    | `$ai-border-strong` |
+|                 | background color | `$field` \*         |
+| AI slug         | size             | mini                |
```
Comment (aagonzales) [inline src/pages/components/checkbox/style.mdx:136]: Checkbox actually has none of these styles, its only modification is the slug.
```suggestion
| --------------- | ---------------- | ------------------- |
| AI slug         | size             | mini                |
```

File: src/pages/components/checkbox/style.mdx [hunk 2]
```diff
@@ -101,3 +118,38 @@ labels should not exceed three words.
   Structure and spacing measurements for vertical and horizontal checkbox
   groupings | px / rem
 </Caption>
+
+## AI presence
+
+The following are the unique styles applied to the components when the AI slug
+is present. Unless specified, all other tokens in the components remain the same
+as the non-AI variants.
```
Comment (aagonzales) [inline src/pages/components/checkbox/style.mdx:126]:
```suggestion
The only style modification an AI variant of the checkbox has is the addition of the slug. All other tokens in the component remain the same as the non-AI variants.
```

File: src/pages/components/radio-button/style.mdx [hunk 3]
```diff
@@ -99,3 +116,38 @@ a phrase and any proper nouns capitalized.
...
+
+More information about designing for AI guidelines is coming soon.
+
+| Element         | Property         | Token / Size        |
+| --------------- | ---------------- | ------------------- |
+| Linear-gradient | start            | `$ai-aura-start-sm` |
+|                 | stop             | `$ai-aura-stop`     |
+| Field           | border-bottom    | `$ai-border-strong` |
+|                 | background color | `$field` \*         |
+| AI slug         | size             | mini                |
```
Comment (aagonzales) [inline src/pages/components/radio-button/style.mdx:134]: AI radio button does not have any of these styles. 

```suggestion
| AI slug         | size             | mini                |
```

File: src/pages/components/radio-button/style.mdx [hunk 4]
```diff
@@ -99,3 +116,38 @@ a phrase and any proper nouns capitalized.
 <Caption>
   Structure and spacing measurements for radio button | px | rem
 </Caption>
+
+## AI presence
+
+The following are the unique styles applied to the components when the AI slug
+is present. Unless specified, all other tokens in the components remain the same
+as the non-AI variants.
```
Comment (aagonzales) [inline src/pages/components/radio-button/style.mdx:124]:
```suggestion
The only style modification an AI variant of the radio button has is the addition of the slug. All other tokens in the component remain the same as the non-AI variants.
```

File: src/pages/components/radio-button/style.mdx [hunk 5]
```diff
@@ -99,3 +116,38 @@ a phrase and any proper nouns capitalized.
...
+
+| Element         | Property         | Token / Size        |
+| --------------- | ---------------- | ------------------- |
+| Linear-gradient | start            | `$ai-aura-start-sm` |
+|                 | stop             | `$ai-aura-stop`     |
+| Field           | border-bottom    | `$ai-border-strong` |
+|                 | background color | `$field` \*         |
+| AI slug         | size             | mini                |
+
+<Caption>
```
Comment (aagonzales) [inline src/pages/components/radio-button/style.mdx:136]: Remove this caption, no longer needed.

File: src/pages/components/checkbox/style.mdx [hunk 6]
```diff
@@ -101,3 +118,38 @@ labels should not exceed three words.
...
+
+| Element         | Property         | Token / Size        |
+| --------------- | ---------------- | ------------------- |
+| Linear-gradient | start            | `$ai-aura-start-sm` |
+|                 | stop             | `$ai-aura-stop`     |
+| Field           | border-bottom    | `$ai-border-strong` |
+|                 | background color | `$field` \*         |
+| AI slug         | size             | mini                |
+
+<Caption>
```
Comment (aagonzales) [inline src/pages/components/checkbox/style.mdx:138]: Remove this caption, no longer needed.

**Review by aagonzales — COMMENTED** [review]
Some changes to the style tab content. Checkbox and radio button are pretty simple from an AI styling perspective. They only have the slug and no additional styles to callout.

**Review by aagonzales — APPROVED** [review]
Looks good to go!

**Review by guidari — APPROVED** [review]

**Pull request merged**